<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QWidget" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>771</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CHAT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogIn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Modern</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Iniciar Sesi√≥n</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSigIn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>320</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Modern</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Registro</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Mono SemiBold</family>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Registro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_NombreR">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_UserR">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PassR">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Registrarme">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Modern</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REGISTRARME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Full name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Back">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>480</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Modern</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <zorder>btn_Back</zorder>
    <zorder>label_2</zorder>
    <zorder>LE_NombreR</zorder>
    <zorder>LE_UserR</zorder>
    <zorder>LE_PassR</zorder>
    <zorder>btn_Registrarme</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Mono SemiBold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Iniciar Sesion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_UserI">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PassI">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_AceptarI">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>380</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Modern</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>aceptar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiBold</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>WELCOME </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Hablar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>HABLAR</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
